src/views/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -16,6 +17,53 @@
   export let projectId: string;
 
   let collapsed = false;
+  let selection: Selection | undefined = undefined;
+
+  onMount(() => {
+    window.addEventListener("keydown", e => {
+      if (e.key === "Escape") {
+        updateHash("");
+      }
+    });
+    window.addEventListener("hashchange", updateSelection);
+
+    return () => window.removeEventListener("hashchange", updateSelection);
+  });
+
+  if (file.diff.type === "plain" && file.diff.hunks.length > 0) {
+    updateSelection();
+    for (const hunkIdx in file.diff.hunks) {
+      const hunk = file.diff.hunks[hunkIdx];
+      for (const lineIdx in hunk.lines) {
+        if (shouldScroll(hunkIdx, lineIdx)) {
+          setTimeout(() => {
+            const target = document.getElementById(
+              file.path + "::H" + hunkIdx + "L" + lineIdx,
+            );
+            if (target) {
+              target.scrollIntoView();
+            }
+          }, 200);
+        }
+      }
+    }
+  }
+
+  function updateSelection() {
+    const fragment = window.location.hash.substr(1);
+    const match = fragment.match(/(.+):H(\d+)L(\d+)(H(\d+)L(\d+))?/);
+    if (match && match[1] === file.path) {
+      selection = {
+        file: match[1],
```

Comment:
This property is never used.

Reply:
https://github.com/radicle-dev/radicle-interface/pull/942/commits/8fcea773f93446c084dc40273d8db4f6f0c7bcc5